reproduciblebuilds/Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM golang 
+FROM golang:1.20 as builder
```

Comment:
Our official documentation says go 1.19, we should use same version as the official doc or update official doc to 1.20.